docs/Systems.md

```diff
@@ -1,5 +1,42 @@
 ## Systems
 
+## Using a Nerves System
+**Single Target**
+
+To use a Nerves system in project with a single target, you can directly include it as part of your application dependencies.
+
+```elixir
+# mix.exs
+
+def deps do
+  [{:nerves_system_bbb, ">=0.0.0"}]
+end
+```
+
+**Multi Target**
+
+Multi target configurations can be handled a number of ways. You can allow all `nerves_system_*` projects by interpolating `target`
+```elixir
+def deps(target) do
+  [{:"nerves_system_#{target}", ">=0.0.0"}]
+end
+```
+
+Or you could switch between different configurations of the same target.
+```elixir
+def deps(:dev = env) do
+  [{:my_custom_bbb_dev, ">=0.0.0"}]
+end
+
+def deps(:prod = env) do
+  [{:my_custom_bbb_prod, ">=0.0.0"}]
+end
+```
+
+Since its all done in Elixir, you can choose which configuration works best for you. Just be sure that there is only ever one `system` and one `toolchain` present when compiling your Nerves application.
```

Comment:
Do you need to worry about only having one `toolchain` present if you only have one `system`? The rule is that only systems list a toolchain in their deps, right?